client/src/hooks/datafiles/mutations/useTrash.ts

```diff
@@ -0,0 +1,127 @@
+import { useSelector, useDispatch, shallowEqual } from 'react-redux';
+import { useMutation } from '@tanstack/react-query';
+import { useSelectedFiles } from 'hooks/datafiles';
+import Cookies from 'js-cookie';
+
+export async function trashUtil({
+  api,
+  scheme,
+  system,
+  path,
+  homeDir,
+}: {
+  api: string;
+  scheme: string;
+  system: string;
+  path: string;
+  homeDir: string;
+}) {
+  const url = `/api/datafiles/${api}/trash/${scheme}/${system}/${path}/`;
```

Comment:
This will need to be converted to use the axios comments that you have down below see 
https://github.com/TACC/Core-Portal/blob/51e2230f3fca51a1e39104b3a7216d0d154d8f38/client/src/hooks/datafiles/mutations/useUpload.ts#L30

Reply:
I just pushed up the changes.

Reply:
@ line 19 we would like to ensure that we return a promise. I'm assuming the promise structure is still the same
https://github.com/TACC/Core-Portal/blob/51e2230f3fca51a1e39104b3a7216d0d154d8f38/client/src/hooks/datafiles/mutations/useUpload.ts#L19